docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,183 @@
+----
+
+Istio Sidecar AutoInjection is done automatically when you label the coherence namespace with istio-injection.
+
+After the namespace is labeled, you can install the operator using your preferred method in the Operator https://oracle.github.io/coherence-operator/docs/latest/#/installation/01_installation[Installation Guide].
+
+After installed operator, use the following command to confirm the operator is running:
+
+[source,bash]
+----
+kubectl get pods -n coherence
+
+NAME                                                     READY   STATUS    RESTARTS   AGE
+coherence-operator-controller-manager-7d76f9f475-q2vwv   2/2     Running   1          17h
+----
+
+2/2 in READY column means that there are 2 containers running in the operator Pod. One is Coherence operator and the other is Envoy Proxy.
+
+== Creating a Coherence cluster with Istio
+
+You can configure your cluster to run with Istio automatic sidecar injection enabled. Before creating your cluster, create the namespace in which you want to run the cluster and label it for automatic injection.
+
+[source,bash]
+----
+kubectl create namespace coherence-example
+kubectl label namespace coherence-example istio-injection=enabled
+----
+
+There is no other requirements to run Coherence in Istio environment.
+
+The following is an example that creates a cluster named example-cluster-storage:
+
+example.yaml
+[source,bash]
+----
+# Example
+apiVersion: coherence.oracle.com/v1
+kind: Coherence
+metadata:
+  name: example-cluster-storage
+----
+
+[source,bash]
+----
+$ kubectl -n coherence-example apply -f example.yaml
+----
+
+After you installed the Coherence cluster, run the following command to view the pods:
+
+[source,bash]
+----
+$ kubectl -n coherence-example get pods
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+example-cluster-storage-0                        2/2     Running   0          45m
+example-cluster-storage-1                        2/2     Running   0          45m
+example-cluster-storage-2                        2/2     Running   0          45m
+----
+
+You can see that 3 members in the cluster are running with 3 pods. 2/2 in READY column means that there are 2 containers running in each Pod. One is Coherence member and the other is Envoy Proxy.
+
+== TLS
+
+Coherence client can support TLS through Istio secure gateway with TLS termination to connect to Coherence cluster running inside kubernetes.  For example, you can apply the following Istio gateway and and virtual service in the namespace of the Coherence cluster:
+
+tlsGateway.yaml
+[source,bash]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: tlsgateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default ingress gateway
+  servers:
+  - port:
+      number: 8043
+      name: tls
+      protocol: TLS
+    tls:
+      mode: SIMPLE
+      credentialName: "extend-credential" # must be the same as secret for the TLS key and cert
```

Comment:
What is this? Who creates this secret, what is in it? Does it also need to be mounted into the Coherence cluster Pods, if so how?

Reply:
Add example to create the secret and server.jks to be used by the Extend client.